### PR TITLE
Added handling of DateOfOccurrence step

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## CLX-483

- Added handling of date of occurrence step

## Checklist

- [x] Dark/light mode verification
- [x] Landscape verification
- [x] iPad verification
- [x] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift [hunk 1]
```diff
@@ -86,40 +86,21 @@ public class ClaimJourneys {
...
             style: .default
         ) {
             action in
             if case .setNewDate = action {
                 PopJourney()
             } else if case .dissmissNewClaimFlow = action {
                 PopJourney()
-            }
-        }
-        .setScrollEdgeNavigationBarAppearanceToStandard
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Journeys/ClaimJourneys.swift:105]: same

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift [hunk 2]
```diff
@@ -86,40 +86,21 @@ public class ClaimJourneys {
...
-            action in
-            if case .dissmissNewClaimFlow = action {
-                PopJourney()
             } else if case .setSingleItemPurchaseDate = action {
                 PopJourney()
+            } else {
+                getScreen(for: action)
             }
         }
         .setScrollEdgeNavigationBarAppearanceToStandard
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Journeys/ClaimJourneys.swift:106]: same

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift [hunk 3]
```diff
@@ -258,15 +239,16 @@ public class ClaimJourneys {
         .hidesBackButton
         .setScrollEdgeNavigationBarAppearanceToStandard
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Journeys/ClaimJourneys.swift:240]: same

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift [hunk 4]
```diff
@@ -86,40 +86,21 @@ public class ClaimJourneys {
...
```
[the remark this answers is not in the document]
Comment (sladan-hedvig) [reply Projects/Claims/Sources/Journeys/ClaimJourneys.swift:86]: Removed :)

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/Claims/Sources/Journeys/ClaimJourneys.swift:105]: Also removed

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift (hunk 2 above)
Comment (sladan-hedvig) [reply Projects/Claims/Sources/Journeys/ClaimJourneys.swift:106]: removed

File: Projects/Claims/Sources/Journeys/ClaimJourneys.swift (hunk 3 above)
Comment (sladan-hedvig) [reply Projects/Claims/Sources/Journeys/ClaimJourneys.swift:240]: remvoed

**Pull request merged**